traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/AnyProductMappingFactory.java

```diff
@@ -36,6 +37,9 @@ public Product instantiate(JsonReader reader) throws Exception {
                 case "purchaseStrategy":
                     purchaseStrategy = reader.nextString();
                     break;
+                case "cancellationPolicy":
```

Comment:
@benRmc @OmarPadierna I don't believe iOS SDK is reading cancellationPolicy, can you guys add?

Reply:
@benRmc Can you look into this?